iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/AffMatchingJob.java

```diff
@@ -112,12 +118,31 @@ private static AffMatchingService createAffMatchingService(JavaSparkContext spar
         AffOrgMatcher firstWordsHashBucketMatcher = createFirstWordsHashBucketMatcher();
         
         
+        affMatchingService.setAffOrgMatchers(of(docOrgRelationMatcher, mainSectionHashBucketMatcher, firstWordsHashBucketMatcher));
         
         
-        affMatchingService.setAffOrgMatchers(of(docOrgRelationMatcher, mainSectionHashBucketMatcher, firstWordsHashBucketMatcher));
+        AffMatchOrganizationAltNameFiller altNameFiller = createAffMatchOrganizationAltNameFiller();
+        affMatchingService.setAffMatchOrganizationAltNameFiller(altNameFiller);
         
         return affMatchingService;
     }
     
+    private static AffMatchOrganizationAltNameFiller createAffMatchOrganizationAltNameFiller() throws IOException {
+        
+        AffMatchOrganizationAltNameFiller altNameFiller = new AffMatchOrganizationAltNameFiller();
```

Comment:
It's only after I read this name for the 10th time I finally realized it is `Filler` not `Filter`. I was going to ask what do we filter here ;)